cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -303,6 +294,11 @@
     &_trigger {
         display: none;
     }
+
+    // JS isn't available to remove u-hidden class, so override it.
+    &_content.u-hidden {
+      display: block;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/3d4d642051a9befc68b2f98d92f696c48849c909